### PR TITLE
index page added

### PR DESCRIPTION
**Pull request by sudharsana-kjl**

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
+body{
+          background-color: #ffebee;
+
+        }
```
Comment (Mindstormer619) [inline public/css/style.css]: Consistency in brackets and spacing please.

File: public/index.php [hunk 2]
```diff
@@ -26,3 +26,12 @@
 */
 
 $app->run();
+
+?>
+
```
Comment (Mindstormer619) [inline public/index.php]: Eliminate blank lines at the end of the file. Keep just one. That is:

```text
Line n-1 : ?>
Line n   :
```

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    <div class="row">
+  <div class="card col s3 l3" >
+    <div class="card red" id="leaderBoard" ><h5>Leader Board</h5></div>
+    <h5><div class="card red btn dept"><a href="#modal1">1</a></div> CSE <a >00</a></h5>
+    <div class="divider"></div>
+    <h5><div class="card red btn dept" >1</div> CSE <a >00</a></h5>
+    <div class="divider"></div>
+    <h5><div class="card red btn dept" >1</div> CSE <a >00</a></h5>
+    <div class="divider"></div>
+    <h5><div class="card red btn dept" >1</div> CSE <a >00</a></h5>
```
Comment (Mindstormer619) [inline public/index.html]: Question: Can you ensure that the script for replacing the placeholders will _definitely_ load before the placeholders are seen? Else it makes for a weird frontend experience.

If there's already code for this, please point me to it.

**Comment by rohithbalaji123** [comment]
Pull Request has been merged with dev-frontend branch.

**Comment by sudharsana-kjl** [comment]
:)

**Pull request closed without merging**